metron-interface/metron-rest/src/main/java/org/apache/metron/rest/controller/SearchController.java

```diff
@@ -45,4 +49,18 @@
   ResponseEntity<SearchResponse> search(final @ApiParam(name = "searchRequest", value = "Search request", required = true) @RequestBody SearchRequest searchRequest) throws RestException {
     return new ResponseEntity<>(searchService.search(searchRequest), HttpStatus.OK);
   }
+
```

Comment:
Feels like any asynchronicity should be at the service layer anyway for other things to potentially benefit from, of course that would bubble up here, but do we really expect the REST server to have to worry about high load?